src/style/app-fixed.module.css

```
@@ -4109,6 +4210,316 @@ form label,
+}
+
+.peopleTabUserTagTableHeaderCell {
+  padding: 0.75rem 1.5rem;
+  text-align: left;
+  font-size: 0.75rem;
+  font-weight: 500;
+  color: var(--peopleTabUserTagTableHeaderCell-color);
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  height: 4rem;
+  font-weight: bold;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-weight is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)